content/events/2021-04-29-developer-call-real-time-communications-on-urbit.md

```diff
@@ -0,0 +1,26 @@
++++
+title = "Developer Call: Real-Time Communication on Urbit"
+date = 2021-04-29
+[extra]
+pinned = false
+host = "Josh Lehman (~wolref-podlex)"
+time = "09:00 PST"
+location = "Online Event"
+image = ""
+registration_url = "https://www.meetup.com/urbit-sf/events/277674273/"
+guests = "Edward Amsden (~ritpub-sipsyl)"
+description = "Video chat on Urbit? It's nearly here with WebRTC."
+youtube = "XFct6JZHPC0"
++++
+
+Video chat on Urbit? It's nearly here using the WebRTC protocol. Edward Amsden (~ritpub-sipsyl) will be giving us a look at the WebRTC Airlock library that he has developed that handles all the hard parts of WebRTC. This will allow developers to simply call or answer calls by ship or app name. Edward will also be giving us a live text-based demo of WebRTC on urbit. Please join us for an exciting presentation.
+
+Edward is a software developer and the principal of Black River Software LLC. He has several years experience building both backend and IoT software in Haskell, and in particular building IoT control and software deployment infrastructure. He also has academic background in programming language research.
+
+As usual, there'll be an opportunity to jump on the call with us afterward to ask questions and hang out.
+
+
+## Resources 
+
+- [Original mailing list discussion of 3rd party software distribution](https://groups.google.com/a/urbit.org)
+- [ucal](https://github.com/taalhavras/ucal)
```

Comment:
Please update with his GitHub project link.

Reply:
Also, might be good to note that he does progress reports in this group: `~ritpub-sipsyl/webrtc-on-urbit`